llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -616,9 +324,10 @@ void saveModuleIR(Module &M, StringRef OutFilename) {
   PrintModule.run(M);
 }
 
-void saveModuleProperties(Module &M, const EntryPointGroup &ModuleEntryPoints,
+void saveModuleProperties(module_split::ModuleDesc &ResM,
```

Comment:
Do we change `ModuleDesc` here? If not (I can find where), should the reference be `const`?

Reply:
We don't change it, but `MAM.getResult<SYCLKernelParamOptInfoAnalysis>(Module &M)` is called here which uses standard `ModuleAnalysisManager` interface.